### DISCUSSION TITLE
No html autocompletion in .html.twig

### DISCUSSION THREAD

**Discussion by ghost** [apache/netbeans#4380] [Q&A]
How can I fix that? Any configuration or ...?

**Comment by mbien** [comment]
I don't think NetBeans supports twig templates.

you could try to create a new file association under `options -> miscellaneous -> files` which maps `twig` to `text/html` but this would break fast since templates are usually no valid html.

**Reply by Chris2011** [reply]
Of course NetBeans can handle twig files out of the box for years. Problem is that it is not fully implemented for mixed HTML/Twig stuff. This is always a big problem inside NetBeans using mixed languages with full code completion support for both (or more) languages.
